packages/calcite-components/src/components/input-date-picker/input-date-picker.e2e.ts

```diff
@@ -805,4 +805,24 @@ describe("calcite-input-date-picker", () => {
       expect(await getFocusedElementProp(page, "id")).toBe("next-sibling");
     });
   });
+
+  it("should reset input value", async () => {
+    const page = await newE2EPage();
+    const expectedValue = "2022-10-01";
+    const expectedInputValue = "10/1/2022";
+
+    await page.setContent(html` <calcite-input-date-picker value="${expectedValue}"></calcite-input-date-picker>`);
+
+    const inputDatePickerEl = await page.find("calcite-input-date-picker");
+    const input = await page.find("calcite-input-date-picker >>> calcite-input");
+
+    expect(await inputDatePickerEl.getProperty("value")).toEqual(expectedValue);
+    expect(await input.getProperty("value")).toEqual(expectedInputValue);
```

Comment:
nitpick: I feel like what we want to assert here is the actual displayed value in the input instead of `calcite-input`'s `value` property, so I've used code like this to do just that:

```
const getInputValue = async () =>
        await page.evaluate(
          () =>
            document
              .querySelector("calcite-input-date-picker")
              .shadowRoot.querySelector("calcite-input")
              .shadowRoot.querySelector("input").value
        );

      expect(await getInputValue()).toBe(expectedInputValue);
```
We could probably create this as a shared util at some point.

Reply:
That would be testing the internals of the `calcite-input` and not the internals of just the `calcite-input-date-picker`. I think its out of scope to test the internals of the input here.

Reply:
Assuming the `this.value` of `calcite-input` 100% matches the input value, which I believe in the case of `calcite-input type="text"` would apply, then testing `calcite-input.value` should suffice.  I agree with you that its testing the internals of `calcite-input` too, and probably worth writing a few tests that capture different localized string values within `calcite-input.e2e.ts` as well.  Definitely out of scope for this PR, but can be a separate nice-to-have test enhancement.